singer/logger.py

```diff
@@ -3,43 +3,10 @@
 import os
 
 
-def get_logger():
-    """Return a Logger instance appropriate for using in a Tap or a Target."""
-    this_dir, _ = os.path.split(__file__)
-    path = os.path.join(this_dir, 'logging.conf')
-    # See
-    # https://docs.python.org/3.5/library/logging.config.html#logging.config.fileConfig
-    # for a discussion of why or why not to set disable_existing_loggers
-    # to False. The long and short of it is that if you don't set it to
-    # False it ruins external module's abilities to use the logging
-    # facility.
-    logging.config.fileConfig(path, disable_existing_loggers=False)
-    return logging.getLogger()
+def get_logger(name='singer'):
+    """Return a Logger instance to use in singer."""
+    if 'LOGGING_CONF_FILE' in os.environ and os.environ['LOGGING_CONF_FILE']:
+        path = os.environ['LOGGING_CONF_FILE']
+        logging.config.fileConfig(path, disable_existing_loggers=False)
 
-
-def log_debug(msg, *args, **kwargs):
```

Comment:
Why do we remove all of these?

Reply:
They are redundant, every file declares a logger instance, so it can be used directly to log info, debug ...etc.

Reply:
👌 Just need to make sure that we don't use it in any repo, otherwise there will be an exception

Reply:
Yes, all taps, targets ...etc would be updated accordingly.